distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -922,14 +922,36 @@ type Column
     ## Parses a text column into values
 
        Arguments:
-       - type: Target type to parse the value to.
-         If `Auto`, will attempt to deduce the type automatically.
-       - format: For date/time types, the format to parse the value with.
-         For boolean types, two values that represent true and false separated
-         by `|`.
-       - on_problems: What to do if the value cannot be parsed.
-         By default, an `Invalid_Format` warning is attached and the value is
-         set to `Nothing`.
+       - type: The type to parse the column to. Defaults to `Auto` meaning that
+         the type will be inferred from the data. If all values fit one of the
+         more specific types, that type is selected. Otherwise the result
+         remains unchanged (a Text column). The types are tried in the following
+         order: Integer, Decimal, Date_Time, Date, Time_Of_Day, Boolean.
+       - format: The formatting settings to use when parsing the column.
+         Defaults to `Nothing` meaning that the default formatting settings will
+         be used. These settings can be overridden by providing a custom
+         `Data_Formatter`. Alternatively, when type is `Date`, `Date_Time`,
+         `Time_Of_Day` or `Boolean`, a text format can be used to specify the
+         format for the chosen datatype. For dates, it is the usual date format
+         notation, and for booleans it should be two values that represent true
+         and false, separated by a `|`.
+       - on_problems: Specifies how to handle if a problem occurs, raising as a
+         warning by default.
+
+       ! Error Conditions
+
+         - If the column is not a text column, an `Invalid_Value_Type` dataflow
+           error is raised.
+         - If some values in the column did not match the expected datatype
+           format, an `Invalid_Format` problem is reported according to the
+           `on_problems` setting. The problematic cells are replaced with
```

Comment:
`according to the `on_problems` setting.` 
I don't think we need to have this. Enough to say `problem is reported.`